platalea/basic.py

```diff
@@ -170,6 +171,9 @@ def val_loss(net):
                 result["validation loss"] = validation_loss
             wandb.log(result)
 
+    # Return loss of the final model for automated testing
+    return {'final_loss': loss_value}
```

Comment:
Is there a reason why we return different criteria (`validation_loss`, `final_loss`) for asr and basic experiments? I would make this consistent by returning the same criteria or both.

Reply:
Or maybe better, do as in (e.g.) `mtl.py` and save and return all intermediate scores.

Reply:
I'll have a look if it can always return the result dict (that is written to json anyway).

Reply:
You might to save the results at each epoch.

Little reminder: please also check `experiments/flickr8k/pip_ind.py`.